inst/tutorials/00-intro/00-intro.Rmd

```diff
@@ -367,10 +367,10 @@ grade_result(
 A `data.frame` is an object in R that is basically like a spreadsheet with some number of rows (units) and some number of columns (variables) and a name for each column. There are a number of ways to interact with a `data.frame` to get useful information about it. For example, if I have a `data.frame` called `mydata`, I can do the following:
 
 - `names(mydata)` - returns the column (variable) names of the data.
-- `ncol(mydata)` - returns the number of columns in the data. 
-- `nrow(mydata)` - returns the number of rows of the data. 
-- `dim(mydata)` - returns a vector of the number of rows and the number of columns (the dimension of the data). 
-- `summary(mydata)` - provides a summary of each variable in the data. 
+- `ncol(mydata)` - returns the number of columns in the data.
+- `nrow(mydata)` - returns the number of rows of the data.
+- `dim(mydata)` - returns a vector of the number of rows and the number of columns (the dimension of the data).
+- `summary(mydata)` - provides a summary of each variable in the data.
 
 These are super useful functions Let's use some of these on a data frame, `UNpop`, which has the same information as the `world.pop` vector, but stored as a data frame.
 
```

Comment:
I think the problem with the colnames is from the following definition of `world.pop` in the `UNpop` data frame. Should be `=` instead of `<-`.

Reply:
Pushed the fixed version.